File: android/src/main/java/org/maplibre/reactnative/components/mapview/MLRNAndroidTextureMapView.java
package org.maplibre.reactnative.components.mapview;

import android.content.Context;
import org.maplibre.android.maps.MapLibreMapOptions;

@SuppressWarnings({"MissingPermission"})
public class MLRNAndroidTextureMapView extends MLRNMapView {
	public static final String LOG_TAG = "MLRNAndroidTextureMapView";
	
    public MLRNAndroidTextureMapView(Context context, MLRNAndroidTextureMapViewManager manager, MapLibreMapOptions options) {
        super(context, manager, options);
    }
}

File: android/src/main/java/org/maplibre/reactnative/components/mapview/MLRNAndroidTextureMapView.kt
package org.maplibre.reactnative.components.mapview

import android.content.Context
import org.maplibre.android.maps.MapLibreMapOptions

class MLRNAndroidTextureMapView(
    context: Context, manager: MLRNAndroidTextureMapViewManager, options: MapLibreMapOptions?
) : MLRNMapView(context, manager, options) {
    companion object {
        const val LOG_TAG: String = "MLRNAndroidTextureMapView"
    }
}
